components/fallback/fallback.cpp

```diff
@@ -22,15 +24,15 @@ namespace Fallback
         if(fallback.empty())
             return 0;
         else
-            return boost::lexical_cast<float>(fallback);
+            return std::stof(fallback);
```

Comment:
It's not the same: g++ test.cpp -o test -std=c++11
```
#include <iostream>
#include <string>

#include <boost/lexical_cast.hpp>

int main()
{
	setlocale(LC_NUMERIC, "de_DE.UTF-8");

	float a = std::stof("1.5");
	float b = std::stof("1,5");

	std::cout << " a" << a << " b " << b << std::endl;

	try
	{
        	a = boost::lexical_cast<float>("1.5");
	}
	catch(...)
	{
		a = 0;
	}
	try
	{
        	b = boost::lexical_cast<float>("1,5");
	}
	catch(...)
	{
		b = 0;
	}

        std::cout << " a" << a << " b " << b << std::endl;

	return 0;
}
```
output:
<pre>
 a1 b 1.5
 a1.5 b 0
</pre>

1. error handling: boost throws an exception, stof silently ignores parsing errors
2. locale: boost::lexical_cast does not use current locale, stof does use it. We don't want to use it. (The default is 'C' usually, but AFAIK QT changes it).